broker-node/app/Http/Controllers/UploadSessionController.php

```diff
@@ -118,6 +124,16 @@ public function update(Request $request, $id)
                     ->where('chunk_idx', $chunk['idx'])
                     ->select('hash')
                     ->first();
+
+                Segment::track([
+                    "userId" => "Oyster",
```

Comment:
Maybe we should use the server's IP as the `userId` ?

Reply:
I would be okay with that.  $_SERVER['REMOTE_ADDR'] doesn't give us the broker's IP though, it gives us the client's.  I will see if there is some global variable we can use that would have the broker's IP.

Reply:
I think 'SERVER_ADDR' will work.

Reply:
SERVER_ADDR did not work.